roles/pulp_redis/tasks/main.yml

```diff
@@ -29,9 +29,18 @@
 
     - name: Install Redis
       package:
-        name: redis
+        name: '{{ pulp_redis_package_name }}'
         state: present
 
+    - name: Enable Redis SCL
+      template:
+        src: templates/redis_scl_profile.j2
+        dest: "/etc/profile.d/redis.sh"
+      when:
+        - ansible_os_family == 'RedHat'
+        - ansible_distribution_major_version|int == 7
+        - pulp_redis_package_name.startswith('rh-')
```

Comment:
Will this cause the SCL to be enabled only if the user is overridding the `pulp_redis_package_name`?

If the answer to ^ is yes, can you confirm you expect after merging this that the user who reruns with the same settings (the defaults) will not have a different redis installed on EL7?

Reply:
Yes to all the above. For this tasks to run one needs:

  * To be on an EL Derivative
  * On EL7 derivative
  * Be using a SCL

If not the above is simply skipped.

I can confirm the above (this is not a package module, just a template module) but either way if you don't specify a SCL this will be skipped @bmbouter

Reply:
Thank you for the explanations. This all sounds good.

Reply:
I agree with this approach as well. Over time, EL7 will be less commonly used. We don't want to over-engineer this with variables (and clutter up the README) unless users specifically request it.